core/config/ocr_config.go

```diff
@@ -0,0 +1,24 @@
+package config
+
+import (
+	"time"
+
+	"github.com/smartcontractkit/chainlink/v2/core/services/keystore/keys/ethkey"
+)
+
+// OCR1Config is a subset of global config relevant to OCR v1.
```

Comment:
This docstring should be: OCR is a subset...

Reply:
Fixed